internal/wrappers/projects-http.go

```diff
@@ -238,3 +263,16 @@ func (p *ProjectsHTTPWrapper) Tags() (
 		return nil, nil, errors.Errorf("response status code %d", resp.StatusCode)
 	}
 }
+
+func setDefaultLimit(params map[string]string) {
```

Comment:
it's not clear.. good idea but it's need a deeper refactoring

Reply:
done